### PR TITLE
(WSR 8) - <Badge/> - remove testkit functions

### PR DESCRIPTION
**Pull request by EgoziE**
Testkit:
- removed `getPrefixIcon` - use your own dataHook on prefix element instead.
- removed `getSuffixIcon` - use your own dataHook on suffix element instead.

### CODE REVIEW

File: src/Badge/Badge.spec.js [hunk 1]
```diff
@@ -105,24 +105,38 @@ describe('Badge', () => {
...
+        expect(content.querySelector(`[data-hook="${dataHook}"]`)).toBeTruthy();
       });
 
       it('should have suffix icon', async () => {
+        const dataHook = 'dataHook';
         const driver = createDriver(
-          <Badge suffixIcon={<Email />}>Hello</Badge>,
+          <Badge
+            suffixIcon={
+              <div data-hook={dataHook}>
```
Comment (shlomitc) [inline src/Badge/Badge.spec.js]: I think you can just pass `data-hook` to the `Email` without the div but not critical

File: src/Badge/Badge.driver.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ const badgeDriverFactory = ({ element, eventTrigger }) => {
   return {
     /** checks if element exists */
     exists: () => !!element,
+    /** returns the element */
+    element: () => element,
```
Comment (shlomitc) [inline src/Badge/Badge.driver.js]: I don't think we need to expose this, why should we?

File: src/Badge/Badge.spec.js (hunk 1 above)
Comment (EgoziE) [reply src/Badge/Badge.spec.js]: for some reason it didn't work, I'll try it again

File: MIGRATION.md [hunk 3]
```diff
@@ -238,6 +239,48 @@ This component was deleted, use `<FormField/>` instead.
   ```
 </details>
 
+## \<Badge/>
+
+This component was deleted, use `<FormField/>` instead.
```
Comment (shlomitc) [inline MIGRATION.md]: That's incorrect

**Pull request merged**